Project/frontend/src/components/Home/index.tsx

```diff
@@ -62,7 +103,17 @@ function Home() {
             disabled={!fileId || isLoading}
             onClick={handleGenerateGraph}
           >
-            {isLoading ? <span className="loading_spinner_home">Generating graph...</span> : "Generate Graph"}
+            {isLoading ? <span className="loading_spinner_home">Working...</span> : "Generate Graph"}
+          </button>
+        </div>
+
+        <div>
+          <button
```

Comment:
FYI, FilePond already includes a built-in remove button (X mark) that uses the onremovefile callback (see the screenshot).
Would it be better to use this built-in button instead of adding a separate delete button? Additionally, the FilePond server property can be customized to handle revert cases. More details are available here: https://pqina.nl/filepond/docs/api/server/#revert-1.
![WhatsApp Image 2024-06-10 at 12 28 42 AM](https://github.com/amosproj/amos2024ss05-knowledge-graph-extractor/assets/166813285/43ced4fe-08f2-4b36-8140-e5f328d8ae7e)